Assets/Scripts/SimpleCard.cs

```diff
@@ -4,20 +4,22 @@
 
 public class SimpleCard : MonoBehaviour
 {
-    public GameObject cardFaceTemplate;
     public GameObject cardFace;
-    public GameObject cardBackTemplate;
 
     public Main mainScript;
```

Comment:
Is this script reference should have public visibility? Guess it will be used only for moving card on scene, so it could be private then

Reply:
agree 👍